### ISSUE TITLE
DRL strategy 

### ISSUE THREAD

**Issue by randheerDas** [AI4Finance-Foundation/FinRL#583]
I have been testing the FinRL_PortfolioAllocation_NeurIPS_2020.ipynb. I have noticed that in most of the runs, the DRL trained strategy is retuning poor results as compared to the DJI Baseline stats. What area should we tune to get the DRL strategy generate better results, like better cumulative returns and higher sharp ratio? (Maybe try a different model) or add new features.

Please advise.

**Comment by zhumingpassional** [comment]
We have not updated it for long time. In the future, we will revise and improve it. Its model is different from others. 

Please see other notebooks currently.

**Comment by rayrui312** [comment]
Basically, there are three directions; algorithm parameters, environment settings, and features. The tuning pipeline may follow like this: 
You can fix a training set, a validation set, and a test set. Train the model on the training set with different algorithm parameters, environment settings, and features. Then check the model's performance on the validation set to select the best one. When you get a satisfying performance on the validation set, you may test your model on the test set.

**Comment by khanhphan1311** [comment]
> 

Thank you for your suggestion!
